### DISCUSSION TITLE
what are the requirements?

### DISCUSSION THREAD

**Discussion by MitkoVtori** [anuraghazra/github-readme-stats#2110] [Q&A] [answered]
Hi, I have two questions about this github stats. Which is better, A or S?
And second, what are the requirements to get A++ and if S is better what are the requirements for it also?

**Comment by rickstaa** [comment, marked as answer]
@MitkoVtori You can find the rank calculation at https://github.com/anuraghazra/github-readme-stats/blob/master/src/calculateRank.js. There are, however, some proposals to improve the ranking since it is really hard to get a different rank than A+ (see https://github.com/anuraghazra/github-readme-stats/issues/455). We will probably merge https://github.com/anuraghazra/github-readme-stats/pull/1186 to fix this in the coming months.
